tests/helper.py

```diff
@@ -2,6 +2,195 @@
 Helper functions for unit tests.
 """
 
+import unittest
+
+import geomstats.backend as gs
+
+
+class TestGeomstatsMethods(unittest.TestCase):
+    def assertScalar(self, result, n_samples=1):
+        return self.assertTrue(
+            gs.allclose(result.shape, (n_samples, 1)),
+            '\nresult.shape = {} instead of {}'.format(
+                result.shape, (n_samples, 1)))
+
+    def assertVector(self, result, n_samples=1, dim=1):
+        return self.assertTrue(
+            gs.allclose(result.shape, (n_samples, dim)),
+            '\nresult.shape = {} instead of {}'.format(
+                result.shape, (n_samples, dim)))
+
+    def assertAllClose(self, result, expected):
+        result = gs.asarray(result)
+        expected = gs.asarray(expected)
+        return self.assertTrue(
+            gs.allclose(result, expected),
+            '\nresult.shape = {}'
+            '\nexpected.shape = {}'
+            '\nresult = {}'
+            '\nexpected = {}'.format(
+                result.shape,
+                expected.shape,
+                result,
+                expected))
+
+    def check_shape_belongs(self, space):
+        point = space.random_uniform()
+        belongs = space.belongs(point)
+        print('belongs={}'.format(belongs))
+
+        self.assertScalar(belongs)
+
+    def check_shape_belongs_vectorization(self, space, n_samples):
+        n_samples = self.n_samples
+
+        points = self.space.random_uniform(n_samples=n_samples)
+        belongs = self.space.belongs(points)
+
+        self.assertScalar(belongs, n_samples)
+
+    def check_shape_random_uniform(self, space, vec_dim):
```

Comment:
not a huge fan of this pattern... Basically this seems like too much logic for tests IMHO.  tests should be clear and super dumb. Here the logic for the testing is itself a bit too complex.